src/pineko/scale_variations.py

```diff
@@ -0,0 +1,266 @@
+"""Module to generate scale variations."""
+import pathlib
+from typing import Tuple
+
+import numpy as np
+import pineappl
+import rich
+from eko import beta
+
+from . import check
+
+AS_NORM = 1.0 / (4.0 * np.pi)
+OrderTuple = Tuple[int, int, int, int]
+"""Tuple representing a PineAPPL order."""
+
+
+def sort_qcd_orders(order: OrderTuple) -> int:
+    """Define the sorting for an order. In particular the order is sorted according to the as order."""
+    return order[0]
+
+
+def ren_sv_coeffs(m, max_as, logpart, which_part, nf):
+    """Return the ren_sv contribution relative to the requested log power and perturbative order contribution (which_part).
+
+    Parameters
+    ----------
+    m : int
+        first non zero perturbative order
+    max_as : int
+        max order of alpha_s
+    logpart : int
+        power of the renormalization scale log asked
+    which_part : int
+        asked perturbative order contribution to be rescaled
+    nf : int
+        number of active flavors
+
+    Returns
+    -------
+    float
+        renormalization scale variation contribution
+    """
+    bcoeff = beta.beta_qcd((max_as - logpart - which_part + 2, 0), nf)
+    as_norm_fact = AS_NORM ** (max_as - which_part)
```

Comment:
```suggestion
    as_normalization = AS_NORM ** (max_as - which_part)
```

`_fact` is fully generic, and if you only need a name different from `as_norm`, then much better to have entire words.

Notice that in principle even `as_norm` would be available, since `as_norm != AS_NORM`. But this would be confusing to refer to.

Reply:
Otherwise, just use a more consistent naming with the previous term, and call it:
```suggestion
    ascoeff = AS_NORM ** (max_as - which_part)
```

Reply:
[2aecc15](https://github.com/NNPDF/pineko/pull/75/commits/2aecc1562cc6a710935adb877ca7cc9156dd2fd9)